WalletWasabi/Helpers/Constants.cs

```diff
@@ -5,80 +5,93 @@
-
-		public const uint ProtocolVersion_WITNESS_VERSION = 70012;
-
-		public const int MaxPasswordLength = 150;
-
-		public static readonly NodeRequirement NodeRequirements = new NodeRequirement
-		{
-			RequiredServices = NodeServices.NODE_WITNESS,
-			MinVersion = ProtocolVersion_WITNESS_VERSION,
-			MinProtocolCapabilities = new ProtocolCapabilities { SupportGetBlock = true, SupportWitness = true, SupportMempoolQuery = true }
-		};
-
-		public static readonly NodeRequirement LocalNodeRequirements = new NodeRequirement
-		{
-			RequiredServices = NodeServices.NODE_WITNESS,
-			MinVersion = ProtocolVersion_WITNESS_VERSION,
-			MinProtocolCapabilities = new ProtocolCapabilities { SupportGetBlock = true, SupportWitness = true }
-		};
-
-		public static readonly NodeRequirement LocalBackendNodeRequirements = new NodeRequirement
-		{
-			RequiredServices = NodeServices.NODE_WITNESS,
-			MinVersion = ProtocolVersion_WITNESS_VERSION,
-			MinProtocolCapabilities = new ProtocolCapabilities
-			{
-				SupportGetBlock = true,
-				SupportWitness = true,
-				SupportMempoolQuery = true,
-				SupportSendHeaders = true,
-				SupportPingPong = true,
-				PeerTooOld = true
-			}
-		};
-
-		public const int P2wpkhInputSizeInBytes = 41;
-		public const int P2pkhInputSizeInBytes = 145;
-		public const int OutputSizeInBytes = 33;
-
-		// https://en.bitcoin.it/wiki/Bitcoin
-		// There are a maximum of 2,099,999,997,690,000 Bitcoin elements (called satoshis), which are currently most commonly measured in units of 100,000,000 known as BTC. Stated another way, no more than 21 million BTC can ever be created.
-		public const long MaximumNumberOfSatoshis = 2099999997690000;
-
-		private static readonly BitcoinWitPubKeyAddress MainNetCoordinatorAddress = new BitcoinWitPubKeyAddress("bc1qs604c7jv6amk4cxqlnvuxv26hv3e48cds4m0ew", Network.Main);
-		private static readonly BitcoinWitPubKeyAddress TestNetCoordinatorAddress = new BitcoinWitPubKeyAddress("tb1qecaheev3hjzs9a3w9x33wr8n0ptu7txp359exs", Network.TestNet);
-		private static readonly BitcoinWitPubKeyAddress RegTestCoordinatorAddress = new BitcoinWitPubKeyAddress("bcrt1qangxrwyej05x9mnztkakk29s4yfdv4n586gs8l", Network.RegTest);
-
-		public static BitcoinWitPubKeyAddress GetCoordinatorAddress(Network network)
-		{
-			Guard.NotNull(nameof(network), network);
-
-			if (network == Network.Main)
-			{
-				return MainNetCoordinatorAddress;
-			}
-
-			if (network == Network.TestNet)
-			{
-				return TestNetCoordinatorAddress;
-			}
-
-			// else regtest
-			return RegTestCoordinatorAddress;
-		}
-
-		public const string ChangeOfSpecialLabelStart = "change of (";
-		public const string ChangeOfSpecialLabelEnd = ")";
-		public const int BigFileReadWriteBufferSize = 1 * 1024 * 1024;
-
-		public const int OneDayConfirmationTarget = 144;
-		public const int SevenDaysConfirmationTarget = 1008;
-	}
+    public static class Constants
+    {
+        public static readonly Version ClientVersion = new Version(1, 1, 6);
```

Comment:
These spaces are some codemaid brainfuck :/ I don't know what the heck is going on.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/1945/commits/000a51437ad3587f5d2ae16b02a4c0781d5e310d

Reply:
I restarted my VS and saved all file again.